code/modules/hydroponics/grown_inedible.dm

```diff
@@ -61,3 +61,28 @@
 	throwforce = 0
 	throw_speed = 4
 	throw_range = 20
+
+/obj/item/weapon/corncob/hollowpineapple
```

Comment:
This typepath is giving me a headache and I'm not sure if this will even be remotely safe - if Bay decides to make corncobs, I'unno, plantable as corn or something, then automatically you will be able to grow corn from pineapples.

This is a contrived example, but point is - you would have to cover all the weird overrides corncobs might have.